packages/mobile/src/screens/upload-screen/fields/ReleaseDateField.tsx

```diff
@@ -0,0 +1,148 @@
+import { useCallback, useMemo, useState } from 'react'
+
+import type { Nullable } from '@audius/common'
+import { Theme } from '@audius/common'
+import { useField } from 'formik'
+import moment from 'moment'
+import { TouchableOpacity } from 'react-native-gesture-handler'
+import type {
+  CustomCancelButtonPropTypes,
+  CustomConfirmButtonPropTypes
+} from 'react-native-modal-datetime-picker'
+import DateTimePickerModal from 'react-native-modal-datetime-picker'
+
+import { Button, Divider, Pill, Text } from 'app/components/core'
+import { makeStyles } from 'app/styles'
+import { useThemeColors, useThemeVariant } from 'app/utils/theme'
+
+const isToday = (date: Date) => {
+  const today = new Date()
+  return (
+    date.getDate() === today.getDate() &&
+    date.getMonth() === today.getMonth() &&
+    date.getFullYear() === today.getFullYear()
+  )
```

Comment:
😅 it really feels like there should be an easier way, but this totally works. No complaints

Reply:
There def is I'll use moment